<?xml version="1.0" ?>

    <!-- this file is auto generated using pedsim_gazebo_plugin pkg -->    
    <sdf version="1.5">
      <world name="default">
      
      <!-- Ground Plane -->
      
      <include>
        <uri>model://ground_plane</uri>
      </include>
    
      <include>
        <uri>model://sun</uri>
      </include>
    
        

     <model name='grey_wall_1'>
      <static>1</static>
      <link name='link'>
      
	  <pose frame=''>-3.0 4.5  1.4 0 0 1.57</pose>
    
        <collision name='collision'>
          <geometry>
            <box>
            
	     <size> 15.0 .2  2.8 </size>

      	   </box>
          </geometry>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode/>
            </contact>
            <bounce/>
            <friction>
              <torsional>
                <ode/>
              </torsional>
              <ode/>
            </friction>
          </surface>
        </collision>
        <visual name='visual'>
          <cast_shadows>0</cast_shadows>
          <geometry>
            <box>
            
	     <size> 15.0 .2  2.8 </size>

      	   </box>
       </geometry>
          <material>
            <script>
              <uri>model://grey_wall/materials/scripts</uri>
              <uri>model://grey_wall/materials/textures</uri>
              <name>vrc/grey_wall</name>
            </script>
          </material>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
    </model>
    

     <model name='grey_wall_2'>
      <static>1</static>
      <link name='link'>
      
	  <pose frame=''>4.5 -3.0  1.4 0 0 0.0</pose>
    
        <collision name='collision'>
          <geometry>
            <box>
            
	     <size> 15.0 .2  2.8 </size>

      	   </box>
          </geometry>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode/>
            </contact>
            <bounce/>
            <friction>
              <torsional>
                <ode/>
              </torsional>
              <ode/>
            </friction>
          </surface>
        </collision>
        <visual name='visual'>
          <cast_shadows>0</cast_shadows>
          <geometry>
            <box>
            
	     <size> 15.0 .2  2.8 </size>

      	   </box>
       </geometry>
          <material>
            <script>
              <uri>model://grey_wall/materials/scripts</uri>
              <uri>model://grey_wall/materials/textures</uri>
              <name>vrc/grey_wall</name>
            </script>
          </material>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
    </model>
    

     <model name='grey_wall_3'>
      <static>1</static>
      <link name='link'>
      
	  <pose frame=''>2.5 12.0  1.4 0 0 0.0</pose>
    
        <collision name='collision'>
          <geometry>
            <box>
            
	     <size> 11.0 .2  2.8 </size>

      	   </box>
          </geometry>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode/>
            </contact>
            <bounce/>
            <friction>
              <torsional>
                <ode/>
              </torsional>
              <ode/>
            </friction>
          </surface>
        </collision>
        <visual name='visual'>
          <cast_shadows>0</cast_shadows>
          <geometry>
            <box>
            
	     <size> 11.0 .2  2.8 </size>

      	   </box>
       </geometry>
          <material>
            <script>
              <uri>model://grey_wall/materials/scripts</uri>
              <uri>model://grey_wall/materials/textures</uri>
              <name>vrc/grey_wall</name>
            </script>
          </material>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
    </model>
    

     <model name='grey_wall_4'>
      <static>1</static>
      <link name='link'>
      
	  <pose frame=''>12.0 2.5  1.4 0 0 1.57</pose>
    
        <collision name='collision'>
          <geometry>
            <box>
            
	     <size> 11.0 .2  2.8 </size>

      	   </box>
          </geometry>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode/>
            </contact>
            <bounce/>
            <friction>
              <torsional>
                <ode/>
              </torsional>
              <ode/>
            </friction>
          </surface>
        </collision>
        <visual name='visual'>
          <cast_shadows>0</cast_shadows>
          <geometry>
            <box>
            
	     <size> 11.0 .2  2.8 </size>

      	   </box>
       </geometry>
          <material>
            <script>
              <uri>model://grey_wall/materials/scripts</uri>
              <uri>model://grey_wall/materials/textures</uri>
              <name>vrc/grey_wall</name>
            </script>
          </material>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
    </model>
    

     <model name='grey_wall_5'>
      <static>1</static>
      <link name='link'>
      
	  <pose frame=''>11.5 15.5  1.4 0 0 0.7853981633974483</pose>
    
        <collision name='collision'>
          <geometry>
            <box>
            
	     <size> 9.899494936611665 .2  2.8 </size>

      	   </box>
          </geometry>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode/>
            </contact>
            <bounce/>
            <friction>
              <torsional>
                <ode/>
              </torsional>
              <ode/>
            </friction>
          </surface>
        </collision>
        <visual name='visual'>
          <cast_shadows>0</cast_shadows>
          <geometry>
            <box>
            
	     <size> 9.899494936611665 .2  2.8 </size>

      	   </box>
       </geometry>
          <material>
            <script>
              <uri>model://grey_wall/materials/scripts</uri>
              <uri>model://grey_wall/materials/textures</uri>
              <name>vrc/grey_wall</name>
            </script>
          </material>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
    </model>
    

     <model name='grey_wall_6'>
      <static>1</static>
      <link name='link'>
      
	  <pose frame=''>15.5 11.5  1.4 0 0 0.7853981633974483</pose>
    
        <collision name='collision'>
          <geometry>
            <box>
            
	     <size> 9.899494936611665 .2  2.8 </size>

      	   </box>
          </geometry>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode/>
            </contact>
            <bounce/>
            <friction>
              <torsional>
                <ode/>
              </torsional>
              <ode/>
            </friction>
          </surface>
        </collision>
        <visual name='visual'>
          <cast_shadows>0</cast_shadows>
          <geometry>
            <box>
            
	     <size> 9.899494936611665 .2  2.8 </size>

      	   </box>
       </geometry>
          <material>
            <script>
              <uri>model://grey_wall/materials/scripts</uri>
              <uri>model://grey_wall/materials/textures</uri>
              <name>vrc/grey_wall</name>
            </script>
          </material>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
    </model>
    

     <model name='grey_wall_7'>
      <static>1</static>
      <link name='link'>
      
	  <pose frame=''>24.5 15.0  1.4 0 0 0.0</pose>
    
        <collision name='collision'>
          <geometry>
            <box>
            
	     <size> 11.0 .2  2.8 </size>

      	   </box>
          </geometry>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode/>
            </contact>
            <bounce/>
            <friction>
              <torsional>
                <ode/>
              </torsional>
              <ode/>
            </friction>
          </surface>
        </collision>
        <visual name='visual'>
          <cast_shadows>0</cast_shadows>
          <geometry>
            <box>
            
	     <size> 11.0 .2  2.8 </size>

      	   </box>
       </geometry>
          <material>
            <script>
              <uri>model://grey_wall/materials/scripts</uri>
              <uri>model://grey_wall/materials/textures</uri>
              <name>vrc/grey_wall</name>
            </script>
          </material>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
    </model>
    

     <model name='grey_wall_8'>
      <static>1</static>
      <link name='link'>
      
	  <pose frame=''>15.0 24.5  1.4 0 0 1.57</pose>
    
        <collision name='collision'>
          <geometry>
            <box>
            
	     <size> 11.0 .2  2.8 </size>

      	   </box>
          </geometry>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode/>
            </contact>
            <bounce/>
            <friction>
              <torsional>
                <ode/>
              </torsional>
              <ode/>
            </friction>
          </surface>
        </collision>
        <visual name='visual'>
          <cast_shadows>0</cast_shadows>
          <geometry>
            <box>
            
	     <size> 11.0 .2  2.8 </size>

      	   </box>
       </geometry>
          <material>
            <script>
              <uri>model://grey_wall/materials/scripts</uri>
              <uri>model://grey_wall/materials/textures</uri>
              <name>vrc/grey_wall</name>
            </script>
          </material>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
    </model>
    

     <model name='grey_wall_9'>
      <static>1</static>
      <link name='link'>
      
	  <pose frame=''>30.0 22.5  1.4 0 0 1.57</pose>
    
        <collision name='collision'>
          <geometry>
            <box>
            
	     <size> 15.0 .2  2.8 </size>

      	   </box>
          </geometry>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode/>
            </contact>
            <bounce/>
            <friction>
              <torsional>
                <ode/>
              </torsional>
              <ode/>
            </friction>
          </surface>
        </collision>
        <visual name='visual'>
          <cast_shadows>0</cast_shadows>
          <geometry>
            <box>
            
	     <size> 15.0 .2  2.8 </size>

      	   </box>
       </geometry>
          <material>
            <script>
              <uri>model://grey_wall/materials/scripts</uri>
              <uri>model://grey_wall/materials/textures</uri>
              <name>vrc/grey_wall</name>
            </script>
          </material>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
    </model>
    

     <model name='grey_wall_10'>
      <static>1</static>
      <link name='link'>
      
	  <pose frame=''>22.5 30.0  1.4 0 0 0.0</pose>
    
        <collision name='collision'>
          <geometry>
            <box>
            
	     <size> 15.0 .2  2.8 </size>

      	   </box>
          </geometry>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode/>
            </contact>
            <bounce/>
            <friction>
              <torsional>
                <ode/>
              </torsional>
              <ode/>
            </friction>
          </surface>
        </collision>
        <visual name='visual'>
          <cast_shadows>0</cast_shadows>
          <geometry>
            <box>
            
	     <size> 15.0 .2  2.8 </size>

      	   </box>
       </geometry>
          <material>
            <script>
              <uri>model://grey_wall/materials/scripts</uri>
              <uri>model://grey_wall/materials/textures</uri>
              <name>vrc/grey_wall</name>
            </script>
          </material>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
    </model>
    

     <model name='grey_wall_11'>
      <static>1</static>
      <link name='link'>
      
	  <pose frame=''>4.5 6.0  1.4 0 0 0.0</pose>
    
        <collision name='collision'>
          <geometry>
            <box>
            
	     <size> 3.0 .2  2.8 </size>

      	   </box>
          </geometry>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode/>
            </contact>
            <bounce/>
            <friction>
              <torsional>
                <ode/>
              </torsional>
              <ode/>
            </friction>
          </surface>
        </collision>
        <visual name='visual'>
          <cast_shadows>0</cast_shadows>
          <geometry>
            <box>
            
	     <size> 3.0 .2  2.8 </size>

      	   </box>
       </geometry>
          <material>
            <script>
              <uri>model://grey_wall/materials/scripts</uri>
              <uri>model://grey_wall/materials/textures</uri>
              <name>vrc/grey_wall</name>
            </script>
          </material>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
    </model>
    

     <model name='grey_wall_12'>
      <static>1</static>
      <link name='link'>
      
	  <pose frame=''>4.5 3.0  1.4 0 0 0.0</pose>
    
        <collision name='collision'>
          <geometry>
            <box>
            
	     <size> 3.0 .2  2.8 </size>

      	   </box>
          </geometry>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode/>
            </contact>
            <bounce/>
            <friction>
              <torsional>
                <ode/>
              </torsional>
              <ode/>
            </friction>
          </surface>
        </collision>
        <visual name='visual'>
          <cast_shadows>0</cast_shadows>
          <geometry>
            <box>
            
	     <size> 3.0 .2  2.8 </size>

      	   </box>
       </geometry>
          <material>
            <script>
              <uri>model://grey_wall/materials/scripts</uri>
              <uri>model://grey_wall/materials/textures</uri>
              <name>vrc/grey_wall</name>
            </script>
          </material>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
    </model>
    

     <model name='grey_wall_13'>
      <static>1</static>
      <link name='link'>
      
	  <pose frame=''>24.0 22.5  1.4 0 0 1.57</pose>
    
        <collision name='collision'>
          <geometry>
            <box>
            
	     <size> 3.0 .2  2.8 </size>

      	   </box>
          </geometry>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode/>
            </contact>
            <bounce/>
            <friction>
              <torsional>
                <ode/>
              </torsional>
              <ode/>
            </friction>
          </surface>
        </collision>
        <visual name='visual'>
          <cast_shadows>0</cast_shadows>
          <geometry>
            <box>
            
	     <size> 3.0 .2  2.8 </size>

      	   </box>
       </geometry>
          <material>
            <script>
              <uri>model://grey_wall/materials/scripts</uri>
              <uri>model://grey_wall/materials/textures</uri>
              <name>vrc/grey_wall</name>
            </script>
          </material>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
    </model>
    

     <model name='grey_wall_14'>
      <static>1</static>
      <link name='link'>
      
	  <pose frame=''>21.0 22.5  1.4 0 0 1.57</pose>
    
        <collision name='collision'>
          <geometry>
            <box>
            
	     <size> 3.0 .2  2.8 </size>

      	   </box>
          </geometry>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode/>
            </contact>
            <bounce/>
            <friction>
              <torsional>
                <ode/>
              </torsional>
              <ode/>
            </friction>
          </surface>
        </collision>
        <visual name='visual'>
          <cast_shadows>0</cast_shadows>
          <geometry>
            <box>
            
	     <size> 3.0 .2  2.8 </size>

      	   </box>
       </geometry>
          <material>
            <script>
              <uri>model://grey_wall/materials/scripts</uri>
              <uri>model://grey_wall/materials/textures</uri>
              <name>vrc/grey_wall</name>
            </script>
          </material>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
    </model>
    

     <model name='grey_wall_15'>
      <static>1</static>
      <link name='link'>
      
	  <pose frame=''>10.0 10.0  1.4 0 0 0.7853981633974483</pose>
    
        <collision name='collision'>
          <geometry>
            <box>
            
	     <size> 2.8284271247461903 .2  2.8 </size>

      	   </box>
          </geometry>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode/>
            </contact>
            <bounce/>
            <friction>
              <torsional>
                <ode/>
              </torsional>
              <ode/>
            </friction>
          </surface>
        </collision>
        <visual name='visual'>
          <cast_shadows>0</cast_shadows>
          <geometry>
            <box>
            
	     <size> 2.8284271247461903 .2  2.8 </size>

      	   </box>
       </geometry>
          <material>
            <script>
              <uri>model://grey_wall/materials/scripts</uri>
              <uri>model://grey_wall/materials/textures</uri>
              <name>vrc/grey_wall</name>
            </script>
          </material>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
    </model>
    

     <model name='grey_wall_16'>
      <static>1</static>
      <link name='link'>
      
	  <pose frame=''>17.0 17.0  1.4 0 0 0.7853981633974483</pose>
    
        <collision name='collision'>
          <geometry>
            <box>
            
	     <size> 2.8284271247461903 .2  2.8 </size>

      	   </box>
          </geometry>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode/>
            </contact>
            <bounce/>
            <friction>
              <torsional>
                <ode/>
              </torsional>
              <ode/>
            </friction>
          </surface>
        </collision>
        <visual name='visual'>
          <cast_shadows>0</cast_shadows>
          <geometry>
            <box>
            
	     <size> 2.8284271247461903 .2  2.8 </size>

      	   </box>
       </geometry>
          <material>
            <script>
              <uri>model://grey_wall/materials/scripts</uri>
              <uri>model://grey_wall/materials/textures</uri>
              <name>vrc/grey_wall</name>
            </script>
          </material>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
    </model>
    

            <plugin name="ActorPosesPlugin" filename="libActorPosesPlugin.so">
        </plugin>
    
    
      </world>
    </sdf>
    
        
